app/controllers/publishers_controller.rb

```diff
@@ -188,6 +188,8 @@ def get_site_banner_data
   private
 
   def authenticate_via_token
+    # For some odd reason, devise flash alerts get displayed during auth
```

Comment:
Can we link to the CustomFailureApp configured on Devise that sets this flash? https://github.com/heartcombo/devise/blob/83a32e6d2118b0535cf54b48df4f9853d85b55fd/lib/devise/failure_app.rb

Reply:
I got rid of the CustomFailureApp since it was causing significant problems related to sign-in.

Reply:
I think that the FailureApp in Devise is called by default, and our CustomFailureApp overrode that functionality.

https://github.com/heartcombo/devise/blob/715192a7709a4c02127afb067e66230061b82cf2/lib/devise/mapping.rb#L123